portal-frontend/src/app/features/notice-of-intents/notice-of-intent-details/parcel/parcel.component.html

```diff
@@ -99,7 +99,7 @@ <h4>Parcel #{{ parcelInd + 1 }}</h4>
     </div>
 
     <ng-container *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">
-      <h4 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government Parcel Contact</h4>
+      <h5 *ngIf="parcel.ownershipTypeCode === PARCEL_OWNERSHIP_TYPES.CROWN">Government Parcel Contact</h5>
       <div class="subheading2 grid-1">First Name</div>
```

Comment:
Do you need to wrap everything below the heading here in `<ng-container *ngIf="parcel.owners[0]">` like you did in application?

Reply:
added